apps/prairielearn/src/schemas/infoCourse.ts

```diff
@@ -91,9 +91,9 @@ export const CourseOptionsJsonSchema = z
     comment: CommentJsonSchema.optional(),
     useNewQuestionRenderer: z
       .boolean()
-      .describe('Feature flag to enable the new question renderer.')
-      .optional()
-      .default(false),
+      .describe('[DEPRECATED, DO NOT USE] Feature flag to enable the new question renderer.')
+      .default(false)
```

Comment:
I don't think there should be a default, if it's missing it should remain missing, right?

Reply:
Addressed in b8850a2.